odoo/addons/base/models/ir_actions.py

```diff
@@ -642,7 +762,43 @@ def _run_action_object_write(self, eval_context=None):
             for field, new_value in res.items():
                 record_cached[field] = new_value
         else:
-            self.env[self.model_id.model].browse(self._context.get('active_id')).write(res)
+            starting_record = self.env[self.model_id.model].browse(self._context.get('active_id'))
+            _, _, target_records = self._traverse_path(record=starting_record)
+            target_records.write(res)
+
+    def _run_action_webhook(self, eval_context=None):
+        """Send a post request with a read of the selected field on active_id."""
+        record = self.env[self.model_id.model].browse(self._context.get('active_id'))
+        url = self.webhook_url
+        if not url:
+            raise UserError(_("I'll be happy to send a webhook for you, but you really need to give me a URL to reach out to..."))
+        vals = {
+            '_model': self.model_id.model,
+            '_id': record.id,
```

Comment:
I want the id to be _always_ included no matter what, hence `_id`
`read` does add `id` by itself, even though I did not ask for it
I did not think worth it to explicitly remove it (esp. since i'd have to check whether the `id` field was not _explicitly_ added to the webhook fields by the user before removing it from the read results)... it didn't seem worth it for something 'technical'

Reply:
true dat